### ISSUE TITLE
Docker projects get stuck in [Building - Build started] state

### ISSUE THREAD

**Issue by tetchel** [eclipse/codewind#1215]
When I deploy PFE using `cwctl`,
```
[19/11/19 21:49:59 pyproj123] [ERROR] containerBuildAndRun : 1361 | Error from server (Forbidden): replicasets.extensions is forbidden: User "system:serviceaccount:codewind:default" cannot list replicasets.extensions in the namespace "codewind": no RBAC policy matched
error executing jsonpath "{.items[0].metadata.uid}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.uid}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}




[19/11/19 21:49:59 pyproj123] [ERROR] containerBuildAndRun : 1361 | 21:49:59 main [ERRO] Must provide <filename> <path_to_update> <value>

[19/11/19 21:49:59 pyproj123] [ERROR] [ File Name: /file-watcher/server/dist/projects/projectUtil.js | Function Name: buildAndRun | Line Number: 1200] The container failed to start for project pyproj123
  Cause: [object Object]
 Error: [object Object]
    at containerBuildAndRun (/file-watcher/server/dist/projects/projectUtil.js:1365:19)
```

lower down,
```
[19/11/19 21:50:03 pyproj123] [ERROR] [ File Name: /file-watcher/server/dist/utils/kubeutil.js | Function Name: getApplicationContainerInfo | Line Number: 97] Failed to get the pod name for: pyproj123
[19/11/19 21:50:03 pyproj123] [ERROR] [ File Name: /file-watcher/server/dist/utils/kubeutil.js | Function Name: getApplicationContainerInfo | Line Number: 98] Error: pods is forbidden: User "system:serviceaccount:codewind:default" cannot list pods in the namespace "codewind": no RBAC policy matched
```

I would guess these RBAC issues are symptoms of another missing piece in the new deployment code, like #1211

**Comment by johnmcollier** [comment]
Looking at the logs the default service account is being used and it hasn't been patched with the necessary roles for Codewind to function. I believe @markcor11 was looking at that today.

#1211 will fix this, though you'll need #1212 as well before deployments work.

**Comment by tetchel** [comment]
ok, i guess this is a dupe of that one then.
